apps/storybook-react-native/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@metamask/storybook-react-native",
```

Add to ensure package isn't published
```suggestion
  "name": "@metamask/storybook-react-native",
  "private": "true", 
```